Add integrations page (#631)
saleor/saleor-docs: docs/developer/extending/integrations.mdx

---
title: Integrations

> description: List of services integrating with Saleor

Reviewer: ```suggestion
description: A list of services integrated with Saleor
```

---

# Integrations

> The following is the list of all the Saleor integrations that are currently being maintained:

Reviewer: ```suggestion
The following list contains all the Saleor integrations that are currently being maintained:
```
